tests/probes_test.go

```diff
@@ -5,11 +5,11 @@ import (
 	"github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
 	v1 "k8s.io/api/core/v1"
-	v13 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
How is this change related to the PR

Reply:
At first I changes on of the tests in probes_tests.go so I fixed those libraries aliases 
v12 and v13 are auto generated names by the IDE

Reply:
As far as I see, now the probles are not part of the PR anymore. The PR is dealing only with the ISCSI network related tests. Can you please remove this commit from the PR. It can be done in a separate PR.

Reply:
Can you please remove this commit from the PR?

Reply:
done